lib/structures/StageInstance.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+const Base = require("./Base");
+
+/**
+* Represents a stage instance
+* @prop {StageChannel} channel The associated stage channel
+* @prop {Boolean} discoverableDisabled Whether or not stage discovery is disabled
+* @prop {Guild} guild The guild of the associated stage channel
+* @prop {String} id The ID of the stage instance
+* @prop {Number} privacyLevel The privacy level of the stage instance. 1 is public, 2 is guild only
+* @prop {String} topic The stage instance topic
+*/
+class StageInstance extends Base {
+    constructor(data, client) {
+        super(data.id);
+        this.client = client;
```

Comment:
This should be this._client (with updated methods) so your console logs don't get drowned by a printed Client.